src/main/java/net/openhft/chronicle/queue/impl/table/AbstractTSQueueLock.java

```diff
@@ -38,15 +39,15 @@
     protected final boolean dontRecoverLockTimeout = Jvm.getBoolean("queue.dont.recover.lock.timeout");
 
     protected final LongValue lock;
-    protected final TimingPauser pauser;
+    protected final CleaningThreadLocal<TimingPauser> pauser;
```

Comment:
Not sure if I should just use normal ThreadLocal here considering none of the TimingPausers have any requirement for cleanup.

Reply:
Yep good question. Might be more intentional to use a normal TL?